settings.py

```diff
@@ -62,6 +62,7 @@ def insert_debug_toolbar_middleware(middlewares):
 # If you're not running on SSL you'll want this to be False.
 SESSION_COOKIE_SECURE = False
 SESSION_COOKIE_DOMAIN = None
+WAFFLE_SECURE = False
```

Comment:
interesting.  Why do we need this?

Reply:
It needs to be `False` so that the cookie works in local dev. Otherwise FF rejects the cookie because of `SameSite`. I'll have to verify how it behaves in our hosted envs.